lib/ae_mdw_web/cont.ex

```diff
@@ -67,6 +67,17 @@ defmodule AeMdwWeb.Continuation do
     end
   end
 
+  def unique_take_and_drop(stream, limit, acc \\ []) do
+    {data, rem_stream} = StreamSplit.take_and_drop(stream, limit)
+    acc = Enum.concat(acc, data) |> Enum.uniq()
```

Comment:
For comprehending, as the `acc` argument is not passed, what case in the future do you think it would be needed? Would you mind adding it later if we are not sure it gonna help soon?

Reply:
it's used in line 75, right?

Reply:
Yes. I mean the argument is not passed to `unique_take_and_drop` so for a first fix maybe we could have the function without the acc parameter if we won't use it soon.

Reply:
For the `Enum.uniq`, on @thepiwo consideration, `Enum.uniq_by` would be faster if we wanna solve the problem firstly in the read.